templates/styles/main.css

```
@@ -1,134 +1,439 @@
+    height: 14rem;
 }
+.first-line {
+    width: 5rem;
+    height: 1.5rem;
+}
+
+.second-line {
+    width: 9rem;
+    height: 1.5rem;
+}
+@-webkit-keyframes skeleton-gradient {
```

Stylelint (17.6.0)

[error] Unexpected vendor-prefixed at-rule "@-webkit-keyframes" (at-rule-no-vendor-prefix)

(at-rule-no-vendor-prefix)